### ISSUE TITLE
EntityCheck Command Error

### ISSUE THREAD

**Issue by mcPr0xy** [Photon-GitHub/AACAdditionPro#151]
Please include:
- Server version: 1.8.8 (spigot)
- ProtocolLib version: Latest (development-build)
- The version of AACAdditionPro: Latest (3.4.0)
- A detailed description of the bug / wanted feature: I tried the new /aacadditionpro entitycheck <player> <duration of check in ticks> command from console and the command does not work, giving me an error.

on 1.8.8

```
29.10 01:16:30 [Server] INFO Error on executing command /aacadditionpro entitycheck mcPr0xy 1000
29.10 01:16:30 [Server] INFO AACAdditionPro - ERROR REPORT - NullPointerException
29.10 01:16:30 [Server] INFO #1: LINE -1 in FILE Z.java (de.photon.AACAdditionPro.Z.c)
29.10 01:16:30 [Server] INFO #2: LINE -1 in FILE X.java (de.photon.AACAdditionPro.X.b)
29.10 01:16:30 [Server] INFO #3: LINE -1 in FILE X.java (de.photon.AACAdditionPro.X.a)
29.10 01:16:30 [Server] INFO #4: LINE -1 in FILE X.java (de.photon.AACAdditionPro.X.a)
29.10 01:16:30 [Server] INFO #5: LINE -1 in FILE Y.java (de.photon.AACAdditionPro.Y.onCommand)
29.10 01:16:30 [Server] INFO #6: LINE 44 in FILE PluginCommand.java (org.bukkit.command.PluginCommand.execute)
29.10 01:16:30 [Server] INFO #7: LINE 141 in FILE SimpleCommandMap.java (org.bukkit.command.SimpleCommandMap.dispatch)
29.10 01:16:30 [Server] INFO #8: LINE 89 in FILE CustomCommandMap.java (gyurix.commands.CustomCommandMap.dispatch)
29.10 01:16:30 [Server] INFO Unknown command. Type "/help" for help.
```

on 1.12.2

```
[02:25:11] [Server thread/WARN]: Unexpected exception while parsing console command "aacadditionpro entitycheck mcPr0xy 5000"
org.bukkit.command.CommandException: Unhandled exception executing command 'aacadditionpro' in plugin AACAdditionPro v3.4.0
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-93e20b3-31d3159]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot.jar:git-Spigot-93e20b3-31d3159]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:651) ~[spigot.jar:git-Spigot-93e20b3-31d3159]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchServerCommand(CraftServer.java:637) [spigot.jar:git-Spigot-93e20b3-31d3159]
	at net.minecraft.server.v1_12_R1.DedicatedServer.aP(DedicatedServer.java:444) [spigot.jar:git-Spigot-93e20b3-31d3159]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:407) [spigot.jar:git-Spigot-93e20b3-31d3159]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot.jar:git-Spigot-93e20b3-31d3159]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot.jar:git-Spigot-93e20b3-31d3159]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_144]
Caused by: java.lang.NullPointerException
	at de.photon.AACAdditionPro.Z.c(Z.java) ~[?:?]
	at de.photon.AACAdditionPro.X.b(X.java) ~[?:?]
	at de.photon.AACAdditionPro.X.a(X.java) ~[?:?]
	at de.photon.AACAdditionPro.X.a(X.java) ~[?:?]
	at de.photon.AACAdditionPro.Y.onCommand(Y.java) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-93e20b3-31d3159]
	... 8 more
```

- If possible a short video of the bug:

**Comment by Janmm14** [comment]
Its the same error, just on 1.8.8 you have some plugin which alteres error output.

**Comment by jiongjionger** [comment]
When I use /aacadditionpro entitycheck, it shows that the player is not online. (paperspigot 1.10.2)

**Comment by Photon-GitHub** [comment]
Addressed in 3.4.1
